setup.py

```diff
@@ -121,7 +129,28 @@ def run_tests(self):
         "Programming Language :: Python :: 3.7",
     ],
     use_scm_version=True,
-    setup_requires=["setuptools_scm"],
+    setup_requires=["setuptools_scm", "cython"],
+    ext_modules=cythonize(
```

Comment:
are these optional?

Reply:
No they are not, you can't build a wheel without those.